package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.5.27",
+  "version": "0.5.29",
```

Comment:
This is correct but you'll get a merge conflict because GH doesn't know whether John's PR is correct or yours.